src/patterns/dosdp-patterns/length_along_axis.yaml

```diff
@@ -0,0 +1,98 @@
+---
+pattern_name: length_along_axis
+
+pattern_iri: http://purl.obolibrary.org/obo/oba/patterns/length_along_axis.yaml
+
+description: "A length trait that is the line along an axis of an anatomical entity."
+
+#  examples:
+#    - http://purl.obolibrary.org/obo/FOVT_0001308  # canine tooth medial-lateral diameter
+#    - http://purl.obolibrary.org/obo/FOVT_0001080  # calcaneous proximal breadth (synonym: calcaneous proximal length medial-lateral)
+
+contributors:
+  - https://orcid.org/0000-0002-7073-9172  # David Osumi-Sutherland
+  - https://orcid.org/0000-0001-8314-2140  # Ray Stefancsik
+  - https://orcid.org/0000-0002-7356-1779  # Nicolas Matentzoglu
+  - https://orcid.org/0000-0003-2699-3066  # Meghan A. Balk
+
+classes:
+  anatomical_entity: UBERON:0001062
+  length: PATO:0000122
+  anatomical_axis: BSPO:0000010
+  taxon: OBI:0100026
+  trait_term: PATO:0000051
+
+relations:
+  characteristic_of: RO:0000052
+  part_of: BFO:0000050
+
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym
+  xref: oio:hasDbXref
+  measured_in_taxon: xsd:string
+  not_measured_in_taxon: xsd:string
+  comment: rdfs:comment
+
+data_list_vars:
+  def_ref: "'xref'"  # reference(s) to the source(s) of definition
+
+vars:
+  element: 'anatomical_entity'
+  axis: 'anatomical_axis'
+  size_trait: 'trait_term'
+  fovt: "xsd:string"  # FOVT id
+  def_ref: "xsd:string"
+  taxa: "taxon"
+  your_comments: "xsd:string"  # comments, clarifications for users
+
+name:
+  text: "%s length"
+  vars:
+    - element
+
+annotations:
+  - annotationProperty: exact_synonym
+    text: "%s length %s"
+    vars:
+      - element
+      - axis
+  - annotationProperty: xref
+    text: "FOVT:%s"  # give FOVT id here
+    vars:
+      - fovt
+  - annotationProperty: measured_in_taxon
+    text: "%s"
+    vars:
+      - taxa
+  - annotationProperty: not_measured_in_taxon
+    text: "%s"
+    vars:
+      - taxa
+
+  - annotationProperty: comment
+    text: "%s"
+    vars:
+      - your_comments
+
+def:
+  text: "The length of a %s along the %s."
+  vars:
+    - element
+    - axis
+  annotations:
+    - annotationProperty: xref
+      value: def_ref
+
+equivalentTo:
+  text: "'length' and ('characteristic_of' some ('part_of' some %s)) and ('follows axis' some %s)"
+  # if part of a bone [e.g., 'distalmost' ('part of' some humerus)]
+  # NOTE: Everything is part of itself.
```

Comment:
It turns out that `BFO:0000050 part of` is actually not reflexive. You can check it by opening `ro.owl` in Protege and checking the Characteristics' pane in the 'Object Properties' tab.

Reply:
should we use "characteristic of part of"?